src/openrct2/rct1/T4Importer.cpp

```diff
@@ -140,7 +140,7 @@ namespace RCT1
             std::string_view vehicleObject;
             if (td4Base.Type == RideType::HedgeMaze)
             {
-                vehicleObject = RCT1::GetRideTypeObject(td4Base.Type);
+                vehicleObject = "rct2.ride.hmaze";
```

Comment:
Why? I know it avoids a lookup, but at least the information on `GetRideTypeObject` is centralized.

Reply:
Why would you do a lookup if you know the _exact_ object it’s going to return? And also, this is down to a quirk in how TD4 works and not RCT1 as a whole - so it makes sense to do it here.